src/autosuggest/__tests__/autosuggest.test.tsx

```diff
@@ -6,6 +6,14 @@ import createWrapper from '../../../lib/components/test-utils/dom';
 import Autosuggest, { AutosuggestProps } from '../../../lib/components/autosuggest';
 import styles from '../../../lib/components/autosuggest/styles.css.js';
 import { expectNoAxeViolations } from '../../__a11y__/axe';
+import { KeyCode } from '@cloudscape-design/test-utils-core/dist/utils';
```

Comment:
`utils` is re-exported at the root so you should be able to import this with 

```
import { KeyCode } from '@cloudscape-design/test-utils-core/utils';
```

Reply:
Thanks, it was inserted by vs-code I think.